### DISCUSSION TITLE
[Feature Request]: Create a planet.getpublii.com

### DISCUSSION THREAD

**Discussion by s3nnet** [GetPublii/Publii#1988] [Ideas]
### Feature Description

With all blogs (partially) dedicated to Publii. Accept my application!

**Comment by bttrx** [comment]
There already is https://getpublii.com/showcase/

**Comment by s3nnet** [comment]
Not really. A Planet (think of Planet KDE) is an open blog network with posts of supporters and contributors about aal sorts of stuff.

**Comment by bttrx** [comment]
> think of Planet KDE

Never heard. So, thanks for bringing this to my attention.

**Comment by bjazmoore** [comment]
Maybe the better solution would be to have an option to integrate an existing blog/website into the Fediverse.  Perhaps this can be done via an addon plugin (This might be a great idea for a premium plugin).

[Developers Guide to ActivityPub](https://thenewstack.io/a-developers-guide-to-activitypub-and-the-fediverse/)
